xarray/backends/common.py

```diff
@@ -369,11 +369,23 @@ class BackendEntrypoint:
     - ``guess_can_open`` method: it shall return ``True`` if the backend is able to open
       ``filename_or_obj``, ``False`` otherwise. The implementation of this
       method is not mandatory.
+    - ``description`` property: it shall return a short string describing the engine,
+    ``""`` otherwise.
+    The implementation of this property is not mandatory.
+    - ``docs`` property: it shall return a string with the URL to the backend's documentation,
+    ``""`` otherwise.
+    The implementation of this property is not mandatory.
     """
 
     open_dataset_parameters: tuple | None = None
     """list of ``open_dataset`` method parameters"""
 
+    description: str = ""
+    """Brief text description of backend"""
+
+    docs: str = ""
+    """URL to backend's documentation"""
+
```

Comment:
The right place to document these class attributes would be in the class' docstring (see https://stackoverflow.com/a/3051356/3154101). Generally you never want to have multiple docstrings (the triple quotes) per class definition or method, because there are tools that assume there is just one per class/method. Class attributes are a slightly weird case.